src/Test/QuickCheck/StateMachineModel/Example.hs

```diff
@@ -104,7 +104,7 @@ semStep prb (Inc ref)     = liftIO $ do
 
 ------------------------------------------------------------------------
 
-debugMem :: MonadIO io => [Untyped MemStep (Ref, Int)] -> io ()
+debugMem :: MonadIO io => [Untyped MemStep (Ref, Pid)] -> io ()
```

Comment:
Yeah, perhaps `debugMem` can be generalised and grouped with the other debugging functions (`ppFork`, `debugShrink`, ...) at the bottom of the other module. Created #26.